contribution.md

```diff
@@ -1,117 +1,135 @@
-# Contribution to Container Images
-
-This document describes rules and best practices for contributing to http://github.com/sclorg/.
+# Contributing to Container Images
 
+This document outlines the guidelines and best practices for contributing to the container images within the [sclorg organization](http://github.com/sclorg/) on GitHub.
 
 ## Pull Requests
 
-All changes done in the images go through pull request (PR) and require review from another maintainer before merging. A reasonably small change that is extra-obvious and cannot cause any issue can be merged/committed without a review (typically typos in documentation).
-
+All modifications to the repositories and images must be submitted via a pull request (PR) and undergo review by another maintainer before merging. Exceptionally, small and obvious changes that pose no risk (such as fixing typos in documentation) may be merged or committed without review.
```

Comment:
```suggestion
All modifications to the repositories and images must be submitted via a Pull Request (PR) and undergo review by another maintainer before merging.
```
Pull Request is in general written with first capital letters. Please check also other occurrences in the text and unify them.